third_party/rust/keccak/v0_1/BUILD.gn

```diff
@@ -33,4 +33,14 @@ cargo_crate("lib") {
   executable_configs += [ "//build/config/compiler:no_chromium_code" ]
   proc_macro_configs -= [ "//build/config/compiler:chromium_code" ]
   proc_macro_configs += [ "//build/config/compiler:no_chromium_code" ]
+  deps = []
+  if (target_cpu == "arm64") {
+    deps += [ "//brave/third_party/rust/cpufeatures/v0_2:lib" ]
+  }
+  if (target_cpu == "x64" || target_cpu == "x86") {
```

Comment:
I don't see this in the keccak Cargo.toml

Reply:
cc: @cdesouza-chromium

Reply:
This is output generated by `gnrt`. I was also confused.